lib/Basic/Targets.cpp

```diff
@@ -6367,7 +6367,8 @@ namespace {
       PtrDiffType = IntPtrType = TargetInfo::SignedInt;
       DescriptionString
         = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-"
-          "v96:128-v192:256-v256:256-v512:512-v1024:1024";
+          "v96:128-v192:256-v256:256-v512:512-v1024:1024"
+          "-n8:16:32:64";
```

Comment:
Argh, I keep forgetting about those tests, sorry… I’ll fix that tonight.
Thanks for pointing that out!